python/spec/dependabot/python/file_updater/pip_compile_file_updater_spec.rb

```diff
@@ -175,17 +175,6 @@
       end
     end
 
-    context "with a met marker that forces a difference Python version" do
-      let(:manifest_fixture_name) { "met_marker.in" }
-      let(:generated_fixture_name) { "pip_compile_met_marker.txt" }
-
-      it "updates the requirements.txt, keeping the unmet dep in it" do
-        expect(updated_files.count).to eq(1)
-        expect(updated_files.first.content).to include("attrs==18.1.0")
-        expect(updated_files.first.content).to include("flaky")
-      end
-    end
```

Comment:
This seems to have mostly been used to distinguish between python 2 and 3 support, so I've yanked the test